### PR TITLE
Translate news 2015-02-23-support-for-ruby-1-9-3-has-ended (zh_cn)

### PR DESCRIPTION
**Pull request by VincentZhao**
Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@hlcfan @pmq20 @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi

Please review this pull request, thanks!

**Comment by hlcfan** [comment]
Suggest to attach the original link in the description, https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

File: zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: "Ruby 1.9.3 的官方支持结束"
+author: "Olivier Lacan"
+translator: "Vincent Zhao"
+date: 2015-02-23 00:00:00 +0000
+lang: zh_cn
+---
+
+从今天开始，对 Ruby 1.9.3 的支持将全部结束。Ruby 新版的 Bug 和安全修复将不会再移植回 1.9.3。
+
+结束维护已经在[一年前预告](https://www.ruby-lang.org/zh_cn/news/2014/01/10/ruby-1-9-3-will-end-on-2015/)。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: @hlcfan you can leave comment here, see [here](https://help.github.com/articles/commenting-on-the-diff-of-a-pull-request/) for more information. In this case we already got a translation so no need to link back to `en` post. :smile:

File: zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: news_post
+title: "Ruby 1.9.3 的官方支持结束"
+author: "Olivier Lacan"
+translator: "Vincent Zhao"
+date: 2015-02-23 00:00:00 +0000
+lang: zh_cn
+---
+
+从今天开始，对 Ruby 1.9.3 的支持将全部结束。Ruby 新版的 Bug 和安全修复将不会再移植回 1.9.3。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: What about the following changes.
**从今天开始，对 Ruby 1.9.3 的支持将全部结束。** => **从今天开始，停止对 Ruby 1.9.3 的全部支持。**
**结束维护已经在** => **停止维护已经在**

File: zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md (hunk 1 above)
Comment (hlcfan) [inline zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: Original link helps the review to make it exact.

File: zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md (hunk 2 above)
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: This change should comment on [L12](https://github.com/ruby/www.ruby-lang.org/pull/1076/files#diff-fa17ad69042c4cb2343a000793c0df77R12) :sweat_smile: 

```
结束维护已经在 => 停止维护已经在
```

Comment (jun1st) [inline zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]:
```
结束维护已经在 => 停止维护已经在
```
支持这个

**Comment by tzwm** [comment]
Looks good for me.

**Comment by VincentZhao** [comment]
Thanks for your review.
I modified the translation of the word "stop". (结束 -> 停止)

> 从今天开始，对 Ruby 1.9.3 的支持将全部结束。 => 从今天开始，停止对 Ruby 1.9.3 的全部支持。

As for this, I think it's missing the subject(主语).
So didn't change.

**Comment by JuanitoFatas** [comment]
Thanks @VincentZhao :wink: :beers:

**Pull request merged**